csrc/validator_utils.h

```diff
@@ -136,6 +136,8 @@ std::pair<double, double> getTolerance(
         return {abs_tol, abs_tol * 0.01};
       }
     }
+    case DataType::Float8_e4m3fn:
+    case DataType::Float8_e5m2:
```

Comment:
@Priya2698 extended the validator to better support fp16. We may need to do something for these new types. I'm not sure if the PR is already merged (is it, @Priya2698?)

Nothing needs to be done for now, but maybe leave a TODO comment here?

Reply:
Are you referring to the regenerated tolerances? (https://github.com/NVIDIA/Fuser/pull/1593). We did not merge this since it wasn't resolving the CI validation errors.